### ISSUE TITLE
Create a Landing Page for Discord Clone.

### ISSUE THREAD

**Issue by PayalKumari10** [Yeasir0032/Discord-Clone#97]
**Description:**
We need a landing page for our Discord clone project that closely mirrors the real Discord landing page. The landing page should be designed using Next.js, TypeScript, Radix UI, and Tailwind CSS. It should also include dark mode support.

**Requirements:**

_1.Tech Stack:_

Next.js,
TypeScript,
Radix UI,
Tailwind CSS.

_2. Landing Page Design:_

The landing page should be an exact copy of the real Discord landing page.
No navigation links should be included on the landing page.

_3. Dark Mode Support:_

The landing page should have dark mode support enabled by default.

**Tasks:**

Set up the Next.js project with TypeScript.
Integrate Radix UI and Tailwind CSS.
Design and implement the landing page to mirror the real Discord landing page.
Add dark mode support.

**Additional Information:**

Ensure the design is responsive and looks good on both desktop and mobile devices.
Adhere to best practices for code structure and organization.
Include comments and documentation where necessary to explain the code and design choices.

**Acceptance Criteria:**

The landing page should be a near-identical copy of the real Discord landing page.
Dark mode should be supported and enabled by default.
No navigation links should be present on the landing page.
The implementation should use Next.js, TypeScript, Radix UI, and Tailwind CSS effectively.

**Comment by karmveershubham** [comment]
I would like to work on this issue under GSSOC'24. Kindly assign this to me.

**Comment by PayalKumari10** [comment]
Hello @karmveershubham,  Sure, go ahead!

**Comment by KanishkChhajed** [comment]
Hello, I am interested to work on this issue, would you assign this issue to me under GSSOC'24.

**Comment by PayalKumari10** [comment]
Hello @karmveershubham Is there any update on this issue? If not, I can assign this issue to others.

**Comment by karmveershubham** [comment]
Yes, assign it to others.

**Comment by PayalKumari10** [comment]
Please Go ahead, @KanishkChhajed .

**Comment by KanishkChhajed** [comment]
Can you tell me that, do I have to add Discord Logo in Nav Bar as their site have or not?

**Comment by PayalKumari10** [comment]
Yes, you can add it to the navbar like the main channel on Discord,, @KanishkChhajed

**Comment by KanishkChhajed** [comment]
Hey, @PayalKumari10  @Yeasir0032, I just create pull request (#99) related to this issue. Please check that request and let me know if there are any updates needed.
